keep-ui/app/(keep)/providers/provider-form.tsx

```diff
@@ -96,52 +100,77 @@ function base64urlencode(a) {
   return btoa(str).replace(/\+/g, "-").replace(/\//g, "_").replace(/=+$/, "");
 }
 
-const DictInput = ({ name, value, onChange, error }) => {
-  const handleEntryChange = (index, field, newValue) => {
-    const newEntries = value.map((entry, i) =>
-      i === index ? { ...entry, [field]: newValue } : entry
+function getConfigsFromArr(arr: [string, ProviderAuthConfig][]) {
```

Comment:
we don't need this function, just use `Object.fromEntries(configs)`;

Reply:
I knew something didn't feel right with that function😅. Thank you.